src/content/docs/en/guides/deploy/github.mdx

```diff
@@ -18,6 +18,32 @@ Astro maintains the official `withastro/action` to deploy your project with very
 
 ## Configure Astro for GitHub Pages
 
+You can deploy your Astro site on GitHub pages with a few configuration steps to ensure that your project can be processed and built successfully.
+
+### Preparing your Astro project for GitHub Pages
+
+By default, GitHub Pages applies Jekyll static site processing, which can interfere with files and folders that begin with an underscore such as Astro's `_astro` directory.
```

Comment:
```suggestion
By default, GitHub Pages applies Jekyll static site processing, which can interfere with files and folders that begin with an underscore such as Astro's `_astro/` directory.
```
We normally add the trailing slash to clearly indicate we're talking about a directory/folder!